[WIP] Dark mode

omniczech opened the pull request:
Fixes #686

Tagged as WIP until folks have had time to give feedback on color choices and review both dark and light modes. Appreciate poking around weird corners of the site as I tried to find every single view but want to be sure.

dekkerglen commented:
![image](https://user-images.githubusercontent.com/28238025/93535453-031bbb80-f915-11ea-885c-52c71a3c3fae.png)
The background of this image doesn't match the banner

![image](https://user-images.githubusercontent.com/28238025/93535470-0dd65080-f915-11ea-86d6-d15031d31884.png)
I think there is too little contrast with this text. Also, the artist name is unreadable.. maybe keep that the way it was before?

![image](https://user-images.githubusercontent.com/28238025/93535551-2e060f80-f915-11ea-8513-6d289a794270.png)
The 'podcast' text is unreadable. The title of these preview objects need more contrast

![image](https://user-images.githubusercontent.com/28238025/93535589-4118df80-f915-11ea-9ec1-341fb80235da.png)
This line looks out of place

![image](https://user-images.githubusercontent.com/28238025/93535630-5726a000-f915-11ea-8fa1-cf76ee9fbb51.png)
I think that the card names should be a little darker, less contrast

![image](https://user-images.githubusercontent.com/28238025/93535671-6d346080-f915-11ea-965a-8191002ef0e0.png)
This lights up when being used

![image](https://user-images.githubusercontent.com/28238025/93535688-7c1b1300-f915-11ea-9e5e-b19523a37f90.png)
This lights up when being used

![image](https://user-images.githubusercontent.com/28238025/93535727-98b74b00-f915-11ea-9cd4-9effa56bdad8.png)
Uh oh!

![image](https://user-images.githubusercontent.com/28238025/93535759-ab318480-f915-11ea-986c-59a0752782a6.png)
React pivot table styles need overwriting

![image](https://user-images.githubusercontent.com/28238025/93535815-c56b6280-f915-11ea-9ae1-444e11ea8f63.png)
Results is not readable

![image](https://user-images.githubusercontent.com/28238025/93535845-d4521500-f915-11ea-84ef-83bae0241ab5.png)
I think this would look better with white text

![image](https://user-images.githubusercontent.com/28238025/93535886-e633b800-f915-11ea-9c77-1b8e137600f8.png)
This vertical line looks wrong

![image](https://user-images.githubusercontent.com/28238025/93535920-f3e93d80-f915-11ea-9e80-87b45bebbf4b.png)
Dark mode scroll bars would be nice 

![image](https://user-images.githubusercontent.com/28238025/93535965-09f6fe00-f916-11ea-9519-689029699f7d.png)
This lights up when being used

![image](https://user-images.githubusercontent.com/28238025/93536013-25620900-f916-11ea-8c4b-8fd7081d33c3.png)
![image](https://user-images.githubusercontent.com/28238025/93536023-2bf08080-f916-11ea-94fb-5801791ca70f.png)
Markdown comment blocks color is incorrect

![image](https://user-images.githubusercontent.com/28238025/93536127-5c381f00-f916-11ea-9177-28eb76147a11.png)
Background color of card names in draftbot analysis is wrong

![image](https://user-images.githubusercontent.com/28238025/93536222-84278280-f916-11ea-8bdb-4795e34c627f.png)
`code` highlighting doesn't have enough contrast with background

omniczech commented:
> ![image](https://user-images.githubusercontent.com/28238025/93535453-031bbb80-f915-11ea-885c-52c71a3c3fae.png)
> The background of this image doesn't match the banner

This is an image where the background is the only area with non-transparent pixels, does this exist in a vector file format to allow us to color the fill on a theme needed basis

> ![image](https://user-images.githubusercontent.com/28238025/93536327-bb962f00-f916-11ea-9f57-fdb5889c5cb8.png)
> I think there is too little contrast with this text. Also, the artist name is unreadable.. maybe keep that the way it was before?

Reverted
 
> ![image](https://user-images.githubusercontent.com/28238025/93536333-c05ae300-f916-11ea-923e-5c15bb20d9de.png)
> The 'podcast' text is unreadable. The title of these preview objects need more contrast

Is there a convenient way to get some demo-data for testing this locally?

> ![image](https://user-images.githubusercontent.com/28238025/93535589-4118df80-f915-11ea-9ec1-341fb80235da.png)
> This line looks out of place

fixed

> ![image](https://user-images.githubusercontent.com/28238025/93535630-5726a000-f915-11ea-8fa1-cf76ee9fbb51.png)
> I think that the card names should be a little darker, less contrast

Brought this down a bit

> ![image](https://user-images.githubusercontent.com/28238025/93535671-6d346080-f915-11ea-965a-8191002ef0e0.png)
> This lights up when being used

The select element has been darkened when focused and on linux firefox this took care of the actual options, but I'd love verification that this isn't overridden in other browsers

> ![image](https://user-images.githubusercontent.com/28238025/93535688-7c1b1300-f915-11ea-9e5e-b19523a37f90.png)
> This lights up when being used

fixed

> ![image](https://user-images.githubusercontent.com/28238025/93535727-98b74b00-f915-11ea-9cd4-9effa56bdad8.png)
> Uh oh!

not sure how my branch was missing the width css, fixed

> ![image](https://user-images.githubusercontent.com/28238025/93535759-ab318480-f915-11ea-986c-59a0752782a6.png)
> React pivot table styles need overwriting

cleaned this up, please check. also removed weird serif font

> ![image](https://user-images.githubusercontent.com/28238025/93535815-c56b6280-f915-11ea-9ae1-444e11ea8f63.png)
> Results is not readable

upped contrast

> ![image](https://user-images.githubusercontent.com/28238025/93535845-d4521500-f915-11ea-84ef-83bae0241ab5.png)
> I think this would look better with white text

hardcoded to always be white

> ![image](https://user-images.githubusercontent.com/28238025/93535886-e633b800-f915-11ea-9c77-1b8e137600f8.png)
> This vertical line looks wrong

fixed 

> ![image](https://user-images.githubusercontent.com/28238025/93535920-f3e93d80-f915-11ea-9e80-87b45bebbf4b.png)
> Dark mode scroll bars would be nice

Is this browser defined? I'm getting dark in my local.

> ![image](https://user-images.githubusercontent.com/28238025/93535965-09f6fe00-f916-11ea-9519-689029699f7d.png)
> This lights up when being used

fixed

> ![image](https://user-images.githubusercontent.com/28238025/93536013-25620900-f916-11ea-8c4b-8fd7081d33c3.png)
> ![image](https://user-images.githubusercontent.com/28238025/93536023-2bf08080-f916-11ea-94fb-5801791ca70f.png)
> Markdown comment blocks color is incorrect

I'm currently getting an error re: the markdown object being undefined after several attempts to `npm i` or even `npm ci`

> ![image](https://user-images.githubusercontent.com/28238025/93536127-5c381f00-f916-11ea-9177-28eb76147a11.png)
> Background color of card names in draftbot analysis is wrong

Where is this page?

> ![image](https://user-images.githubusercontent.com/28238025/93536222-84278280-f916-11ea-8bdb-4795e34c627f.png)
> `code` highlighting doesn't have enough contrast with background

I'm not stoked on the solution I've got in there, but take a look.
